roles/poem/templates/ssl.conf.j2

```diff
@@ -215,8 +218,6 @@ BrowserMatch "MSIE [2-5]" \
 CustomLog logs/ssl_request_log \
           "%t %h %{SSL_PROTOCOL}x %{SSL_CIPHER}x \"%r\" %b"
 
-RewriteEngine On
-RewriteRule ^/$ /poem [R]
```

Comment:
Please, leave RewriteEngine on and rule as it is, because it helps redirecting user to web application URL path when he just enters the URL without path (https://poem.egi.eu). We can get A+ with this lines presented in config.

Reply:
done